src/genai_detection/pipelines/data_processing/nodes.py

```diff
@@ -1,3 +1,68 @@
 import logging
+from kedro.pipeline import Pipeline, pipeline, node
+
+from torchvision import transforms, models
+import torch
+
+from genai_detection.settings import TRAIN_CONFIG
 
 LOGGER = logging.getLogger(__name__)
+
+def transform_raw_image_data(raw_image_dataset):
+    """Preprocess raw image data."""
+    transformation = transforms.Compose([
+        transforms.Resize((256, 256)),
+        transforms.ToTensor(),
+    ])
+    raw_image_dataset.transform = transformation
+    return raw_image_dataset
+
+preprocess_raw_image_data_node = node(
+                func=transform_raw_image_data,
+                inputs="image_test_data",
```

Comment:
I think that should be image_train_data

Edit:
Shouldn't be something like "data/01_raw"?